modules/default.nix

```diff
@@ -16,7 +20,7 @@ let
   rawModule = evalModules {
     modules = [
       {
-        _module.args.home-manager = home-manager;
+        _module.args.home-manager = hmPath;
```

Comment:
If you change what is inside the `home-manager` argument, that could be a breaking change if anybody uses this argument in their configuration. But I don't really see a reason why anybody would do this.

Nonetheless, a name like `home-manager-path` would be better, because nobody would assume that `home-manager` is just a path.